src/components/pages/budget/BudgetList.vue

```diff
@@ -339,14 +495,111 @@ export default {
+            convertedExpenses[departmentId][personId][month] = cost
+            if (!convertedExpenses[departmentId][month]) {
+              convertedExpenses[departmentId][month] = 0
+            }
+            convertedExpenses[departmentId][month] += cost
+            if (!convertedExpenses[month]) {
+              convertedExpenses[month] = 0
+            }
+            convertedExpenses[month] += cost
+            personTotal += cost
+          })
+
+          convertedExpenses[departmentId][personId].total = personTotal
+          departmentTotal += personTotal
+        })
+        convertedExpenses[departmentId].total = departmentTotal
+        total += departmentTotal
+      })
+      convertedExpenses.total = total
+      return convertedExpenses
+    },
+
+    extendedBudgetDepartments() {
+      if (!this.isShowingExpenses) return this.budgetDepartments
+      return this.budgetDepartments
+    },
+
+    differences() {
+      const differences = {}
+      this.budgetDepartments.forEach(department => {
+        if (!differences[department.id]) {
+          differences[department.id] = { total: 0 }
+        }
+        console.log(department.id)
+        department.persons.forEach(person => {
+          if (!person.person_id) return
+          if (!differences[department.id][person.person_id]) {
+            differences[department.id][person.person_id] = 0
+          }
+          let personTotal = 0
+          Object.keys(this.costsMonths).forEach(month => {
+            personTotal += this.getMonthCost(person, month)
+          })
+          const expense =
+            this.convertedExpenses[department.id]?.[person.person_id]?.total ||
+            0
+          const personDifference = personTotal - expense || 0
+          console.log(person.person_id, personDifference, personTotal, expense)
+          differences[department.id][person.person_id] += personDifference
+          differences[department.id].total += personDifference
+        })
+      })
+      return differences
+    }
   },
 
   methods: {
     ...mapActions(['updateProductionBudgetEntry']),
 
+    getDailyRate(deparmentId, personId) {
+      const person = this.personMap.get(personId)
+      if (!person) return 0
+      const salaryScale =
+        person.seniority && person.position
+          ? this.salaryScale[deparmentId][person.position][person.seniority]
+          : 0
+      return person.daily_salary || salaryScale
+    },
+
+    convertTimeSpentToCost(dailyRate, timeSpent) {
+      const days = timeSpent / 60 / this.organisation.hours_by_day
+      return Math.round(days * dailyRate)
+    },
+
     getMonthCost(personEntry, month) {
-      const monthKey = month.format('YYYY-MM')
+      let monthKey = ''
+      if (typeof month === 'string') {
+        monthKey = month
+      } else {
+        monthKey = month.format('YYYY-MM')
+      }
+      personEntry.exceptions = personEntry.exceptions || {}
```

Comment:
duplicate code with the line 603
```suggestion
      personEntry.exceptions = personEntry.exceptions || {}
```